### ISSUE TITLE
add a help/info menu

### ISSUE THREAD

**Issue by dfirebaugh** [dfirebaugh/spritely#5]
When a user presses f1 (or a button in the UI), we should display a help menu that shows keyboard shortcuts.

```
ctrl+c  copy
ctrl+v  paste
left click to draw pixel
right click to select pixel that is on the canvas
```

We will add more later.

**Comment by Satharus** [comment]
Hey man :D 

great project, I love it. 

I fixed this issue and #6 and I will make the pull request tomorrow. 

Thanks!

**Comment by dfirebaugh** [comment]
sweet, thanks!

I will look for your PR/s.

Feel free to offer suggestions on improvements as well.
